wpilibc/src/main/native/include/frc/Compressor.h

```diff
@@ -79,31 +79,39 @@ class Compressor : public wpi::Sendable,
   bool IsEnabled() const;
 
   /**
-   * Check if the pressure switch is triggered.
+   * Returns the state of the pressure switch.
    *
-   * @return true if pressure is low
+   * @return True if pressure switch indicates that the system is not full,
+   * otherwise false.
    */
   bool GetPressureSwitchValue() const;
 
   /**
-   * Query how much current the compressor is drawing.
+   * Get the current drawn by the compressor.
    *
-   * @return The current through the compressor, in amps
+   * @return Current drawn by the compressor.
    */
   units::ampere_t GetCurrent() const;
 
   /**
-   * Query the analog input voltage (on channel 0) (if supported).
+   * If supported by the device, returns the analog input voltage (on channel
+   * 0).
    *
-   * @return The analog input voltage, in volts
+   * This function is only supported by the REV PH. On CTRE PCM, this will
+   * return 0.
+   *
+   * @return The analog input voltage, in volts.
    */
   units::volt_t GetAnalogVoltage() const;
 
   /**
-   * Query the analog sensor pressure (on channel 0) (if supported). Note this
-   * is only for use with the REV Analog Pressure Sensor.
+   * If supported by the device, returns the pressure read by the analog
+   * pressure sensor (on channel 0).
+   *
+   * This function is only supported by the REV PH with the REV Analog Pressure
```

Comment:
Does this not work with any 5V analog pressure sensor?

Reply:
No, because the voltage to pressure scale inside the PH is calibrated to the REV sensor. Other sensors might have a different voltage scale.